quodlibet/quodlibet/ext/events/synchronizedlyrics.py

```diff
@@ -179,78 +215,184 @@ def _style_lyrics_window(self):
         compressed = []
+        compressedWords = []
+        bracketType = 0
+        goToNextLine = True
+        wentToNextLine = False
         while keep_reading:
-            next_find = raw_file.find("[", begin + 1)
-            if next_find == -1:
-                keep_reading = False
-                line = raw_file[begin:]
+            if goToNextLine:
+                next_line = raw_file.find("[", begin + 1)
+                wentToNextLine = True
+            goToNextLine = False
+
+            start_ELRC = raw_file.find("[", beginELRC, next_line)
+            bracketType = 0
+            if start_ELRC == -1:
+                start_ELRC = raw_file.find("<", beginELRC, next_line)
+                bracketType = 1
+            next_ELRC = raw_file.find("<", start_ELRC + 1, next_line)
+
+            if next_ELRC == -1:
+                next_ELRC = next_line
+                goToNextLine = True
+
+            if next_line == -1:
+                start_ELRC = raw_file.find("[", beginELRC)
+                bracketType = 0
+                if start_ELRC == -1:
+                    start_ELRC = raw_file.find("<", beginELRC)
+                    bracketType = 1
+                next_ELRC = raw_file.find("<", start_ELRC + 1)
+                word = raw_file[start_ELRC:next_ELRC]
+                if next_ELRC == -1:
+                    keep_reading = False
+                    word = raw_file[start_ELRC:]
             else:
-                line = raw_file[begin:next_find]
-            begin = next_find
+                word = raw_file[start_ELRC:next_ELRC]
+
+            begin = next_line
+            beginELRC = next_ELRC
 
             # parse lyricsLine
-            if len(line) < 2 or not line[1].isdigit():
+            if len(word) < 2 or not word[1].isdigit():
                 continue
-            close_bracket = line.find("]")
-            t = datetime.strptime(line[1:close_bracket], '%M:%S.%f')
+            if bracketType == 0:
+                close_bracket = word.find("]")
+                line = raw_file[start_ELRC:next_line]
+            if bracketType == 1:
+                close_bracket = word.find(">")
+            t = datetime.strptime(word[1:close_bracket], '%M:%S.%f')
             timestamp = (t.minute * 60000 + t.second * 1000 +
                          t.microsecond / 1000)
-            words = line[close_bracket + 1:]
+            if wentToNextLine:
+                wentToNextLine = False
+                stripELRC = True
+                newLineToStrip = line[close_bracket + 1:]
+                currentStrip = newLineToStrip
+                beginStrip = 0
+                while stripELRC:
+                    startStrippingELRC = newLineToStrip.find("<", beginStrip)
+                    endStrippingELRC = newLineToStrip.find(">",
+                        beginStrip + 1)
+                    strippedLine = currentStrip.replace(
+                    newLineToStrip[startStrippingELRC:endStrippingELRC + 1],
```

Comment:
indentation seems off here

Reply:
Also can we stick to `snake_case` for Python variables